api/v1/rpc_get_join_token.go

```diff
@@ -9,6 +11,8 @@ type GetJoinTokenRequest struct {
 	Name string `json:"name"`
 	// Worker should be set to true to generate a token for joining a worker node.
 	Worker bool `json:"worker"`
+	// TTL is the duration until the token expires (time-to-live).
+	TTL time.Duration `json:"ttl,omitempty"`
```

Comment:
what's the default?

Reply:
Defaults to https://github.com/canonical/k8s-snap/pull/621/files#diff-ed4a14595cce53159213e75b2d4cf0f555dfbece53bd3eb66018adda2d82ac14R33 24 hours. 
Not sure if this is a sane default? (I think so). Will update the comment once we agreed on this value.

Reply:
24h works for me.

Reply:
understood... you are reserving the right to default it in the k8s-snap implementation